std_msgs/Float64 move_distance